packages/livekit-rtc/src/participant.ts

```diff
@@ -266,6 +293,151 @@ export class LocalParticipant extends Participant {
     pub.track = undefined;
     this.trackPublications.delete(trackSid);
   }
+
+  /**
+   * Initiate an RPC request to a remote participant.
+   * @param destinationIdentity - The `identity` of the destination participant
+   * @param method - The method name to call
+   * @param payload - The method payload
+   * @param responseTimeoutMs - Timeout for receiving a response after initial connection
+   * @returns A promise that resolves with the response payload or rejects with an error.
+   * @throws Error on failure. Details in `message`.
+   */
+  async performRpcRequest(
+    destinationIdentity: string,
+    method: string,
+    payload: string,
+    responseTimeoutMs?: number,
```

Comment:
drop "ms" here and everywhere else, node assumes times and durations are in milliseconds by default

Reply:
I'm going to leave this as-is since this feature directly crosses language boundaries I'd like to keep the interface as consistent as possible with the wire protocol. this field will be called `response_timeout_ms` in every SDK.